lib/supply/uploader.rb

```diff
@@ -69,11 +69,29 @@ def upload_screenshots(language)
     def upload_binary
       if Supply.config[:apk]
         Helper.log.info "Preparing apk at path '#{Supply.config[:apk]}' for upload..."
+        apk_version_code = client.upload_apk(Supply.config[:apk])
+
+        Helper.log.info "Updating track '#{Supply.config[:track]}'..."
         if Supply.config[:track].eql? "rollout"
-          client.upload_apk_to_track_with_rollout(Supply.config[:apk], Supply.config[:track], Supply.config[:rollout])
+          client.update_track(Supply.config[:track], Supply.config[:rollout], apk_version_code)
         else
-          client.upload_apk_to_track(Supply.config[:apk], Supply.config[:track])
+          client.update_track(Supply.config[:track], 1.0, apk_version_code)
+        end
+
+        if metadata_path
+          Dir.foreach(metadata_path) do |language|
+            next if language.start_with?('.') # e.g. . or .. or hidden folders
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.